src/components/character-avatar/character-avatar.tsx

```diff
@@ -20,12 +28,38 @@ export const CharacterAvatar: FC<Props> = ({
   isBot,
   onSelectPlayer,
 }) => {
+  const animation = getCharacterAnimation(character.name, isBot, isSelected);
+  const [segments, setSegments] = useState<number[][]>([[]]);
+
+  useEffect(() => {
+    if (isBot) {
```

Comment:
What does `isBot` indicate? That someone voted it locally as a bot or is this data we retrieve from the server? If it's the latter keep in mind that we only return `isBot` from the API only in the results stage since users would be able to cheat otherwise